styles/style.css

```diff
@@ -109,7 +106,6 @@ a img {
 button {
```

Comment:
The width here (line 92) is keeping your site from working on a small screen. It's the same in line 124, and line 231.